vademecum_VT.md

```diff
@@ -0,0 +1,63 @@
+**_Si consiglia di leggere prima il file README.md_**
+
+# Scopri Firefox e Mozilla!  |VT|
+
+##### Una breve presentazione tecnica dei progetti Mozilla
+
+Capitoli:
+* Presentazione
+* Firefox
+* Test Pilot
+* Altri esperimenti
+* Strumenti utilizzati
+* Unisciti a noi. Diventa un volontario Mozilla Italia ora!
+
+### Presentazione
+**Mozilla Foundation** è un'organizzazione no-profit, presente ovunque nel mondo. La comunità italiana si chiama **Mozilla Italia** (**mozillaitalia.orgs**).
+Questa è la versione tecnica, indirizzata perlopiù a un pubblico con conoscenze più specifiche e specializzanti.
+
+### Firefox |QRcode|
+Il browser web sviluppato per offrire un'esperienza di navigazione **unica** e **personalizzata**.
+
+#### Add-ons |QRcode|
+Firefox dispone di migliaia di componenti aggiuntivi, tra estensioni, temi, plugin e altro, con supporto alle WebExtensions.
+
+#### WebVR |QRcode|
+Esplora la realtà virtuale direttamente da Firefox, senza necessità di installare software specifici. Con WebVR la realtà aumentata e virtuale è gratuita, semplice e per tutti.
+
+#### Rust |QRcode|
+Un linguaggio di programmazione sviluppato per offrire un'esperienza unica, ma allo stesso tempo completa e complessa. E' un linguaggio di programmazione famigliare a chi già sa programmare, la sintassi è simile a quella del C.
+
+### Test Pilot |QRcode|
+Mozilla sviluppa numerosi progetti online, uno di essi è Test Pilot, il quale permette di utilizzare funzionalità uniche di Firefox ancora in fase di testing.
+
+#### Send |QRcode|
+Condividi file che automaticamente vengono distrutti dopo il download, il tutto gratuitamente, senza registrazione e con pochi semplici clic.
+
+#### Notes |QRcode|
```

Comment:
togleirei notes che è un test pilot e mettere qualcos'altro, forse IHR

Reply:
Ma la sezione è proprio "Test Pilot", vorrà dire rimuovere proprio questa sezione allora

Reply:
Certo! Non ci avevo pensato, in realtà 😟